client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -442,35 +100,28 @@ def process(self, instance):
 
         assert self._deadline_url, "Requires Deadline Webservice URL"
 
-        file_path = None
-        if self.use_published:
-            if not self.import_reference:
-                file_path = self.from_published_scene()
-            else:
-                self.log.info("use the scene with imported reference for rendering") # noqa
-                file_path = context.data["currentFile"]
-
-        # fallback if nothing was set
-        if not file_path:
-            self.log.warning("Falling back to workfile")
-            file_path = context.data["currentFile"]
-
-        self.scene_path = file_path
-        self.log.info("Using {} for render/export.".format(file_path))
+        job_info = self.get_generic_job_info(instance)
+        self.job_info = self.get_job_info(job_info)
 
-        self.job_info = self.get_job_info()
+        self._set_scene_path(
+            context.data["currentFile"], job_info.UsePublished)
         self.plugin_info = self.get_plugin_info()
+
         self.aux_files = self.get_aux_files()
 
+        plugin_info_data = instance.data["deadline"]["plugin_info_data"]
```

Comment:
Collect Job Info MAY not have triggered if this submission plugins runs over a family that the Collect Job Info does not.

See @MustafaJafar 's issue [here](https://github.com/ynput/ayon-deadline/pull/49#issuecomment-2455217702)
```suggestion
        plugin_info_data = instance.data["deadline"].get("plugin_info_data", {})
```

Reply:
Ignore me - fixed with https://github.com/ynput/ayon-deadline/pull/49/commits/42658bcaee7c6216c86b32473d23d95e8598999b

Reply:
In that case we will have issue much sooner. (when querying `job_info`)